tests/typowalking/wordwalker.js

```diff
@@ -0,0 +1,129 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: nanospell */
+
+'use strict';
+
+bender.editor = {
+	config: {
+		enterMode: CKEDITOR.ENTER_P
+	},
+};
+
+bender.test( {
+	setUp: function() {
+		this.editor = this.editorBot.editor;
+	},
+	getWordsWithWordWalker: function(root) {
+		var editor = this.editorBot.editor,
+			range,
+			wordwalker,
+			wordsReturned = [],
+			currWordObj;
+
+		range = new CKEDITOR.dom.range( editor.document );
+		// assume there is only one block level element.
+		range.selectNodeContents( root );
+
+		wordwalker = new editor.plugins.nanospell.WordWalker(range);
+
+		while (currWordObj = wordwalker.getNextWord()) {
+			wordsReturned.push(currWordObj.word);
+		}
+
+		return wordsReturned;
+	},
+
+	'test walking a simple paragraph': function() {
+		var bot = this.editorBot,
+			wordsReturned;
+		bot.setHtmlWithSelection( '<p>foo bar baz</p>' );
+
+		wordsReturned = this.getWordsWithWordWalker(this.editor.editable().getFirst() );
+
+		arrayAssert.itemsAreEqual(['foo', 'bar', 'baz'], wordsReturned);
+	},
+
+	'test walking a simple paragraph with inline formats': function() {
+		var bot = this.editorBot,
+			wordsReturned;
+		bot.setHtmlWithSelection( '<p>f<i>o</i>o <strong>b</strong>ar <em>baz</em></p>' );
```

Comment:
Agree, I started doing something similar in my newer tests (after this PR).

I avoided it here because I also hate the + at end of each line, but it's probably better than otherwise